doc/api/next_api_changes/2018-06-27-AL.rst

```diff
@@ -1,6 +1,9 @@
 Changes to backend loading
 ``````````````````````````
 
+Assignment to ``rcParams["backend"]`` now sets the backend.  Backends can now
```

Comment:
As discussed in #11612 I'm :-1: on making `rcParams["backend"]` active. This is a new way of API behavior, which we don't want in the long run. There is no loss in functionality, if users still use `matplotlib.use()` to set the backend.

Can we leave that part out in the PR?